README.md

```diff
@@ -72,7 +72,7 @@ $obj = $(Get-WMIObject Win32_UserAccount -Filter "Name='Guest'")
 $obj.Rename("OtherGuest")
 ~~~
 
-This has the added benefit of not requiring escaping '$' in the PowerShell code.
+This has the added benefit of not requiring escaping '$' in the PowerShell code. Note that templates need to have DOS linefeeds or they will not work as expected. One tool for converting UNIX linefeeds to DOS linefeeds is [unix2dos](http://freecode.com/projects/dos2unix).
```

Comment:
`-File` has issues with exit codes - specifically why we avoid it like the plague. We support POSH v2 and forward. If you want some reading to understand the reasoning - http://joshua.poehls.me/2012/powershell-script-module-boilerplate/